server/test/unit/common/test_app_config.py

```diff
@@ -121,10 +122,11 @@ def test_get_config_vars_from_aws_secrets(self, mock_get_secret_key):
 
             from server.common.aws_secret_utils import handle_config_from_secret
             # should not throw error
-            # "AttributeError: 'ServerConfig' object has no attribute 'user_annotations__hosted_tiledb_array__db_uri'"
+            # "AttributeError: 'XConfig' object has no attribute 'x'"
             handle_config_from_secret(config)
 
             # should throw 2 errors (one for each var set from a secret)
-            self.assertEqual(len(logger.output), 2)
+            self.assertEqual(len(logger.output), 3)
             self.assertIn('ERROR:root:set app__flask_secret_key from secret', logger.output[0])
```

Comment:
You can also directly check for success by checking the config, e.g:
```
self.assertEqual(confg.server_config.app__flask_secret_key, "mock_flask_secret")
```

Reply:
oh good call!